_data/toc.yaml

```diff
@@ -22,43 +22,49 @@ guides:
   section:
   - path: /engine/installation/
     title: Install Docker
-  - sectiontitle: Docker Edge
+  - sectiontitle: Docker EE
```

Comment:
I'm mostly agnostic about this but I think we want people to know about Docker EE since it is the new thing. WDYT @friism